overlord/managers_test.go

```diff
@@ -78,13 +87,19 @@ func (ms *mgrsSuite) SetUpTest(c *C) {
 	ms.aa = testutil.MockCommand(c, "apparmor_parser", "")
 	ms.udev = testutil.MockCommand(c, "udevadm", "")
 
+	rootPrivKey, _ := assertstest.GenerateKey(1024)
+	storePrivKey, _ := assertstest.GenerateKey(752)
```

Comment:
this test-only helpers actually panic (it's the style of the fully asserttest package), this is just ignoring a 2nd result that we don't use here